lib/index.js

```diff
@@ -1,29 +1,68 @@
 'use strict';
 
+var pg = require('pg');
 var addDefaultValues = require('./add_default_values.js');
 var loadSchema = require('./load_schema.js');
 var updateSchema = require('./update_schema.js');
+var routeGen = require('./helpers/endpoints_helpers/endpoints_generator.js');
+var configurationRoutes = require('./helpers/configuration/configuration.js');
+var db = require('./db.js');
+
+require('env2')('config.env');
 
 module.exports = function (server, options, next) {
+  var routes;
+
+  if (!process.env.DATABASE_URL) {
+    return next(new Error('DATABASE_URL environment variable is undefined'));
+  }
+
+  // server.app.db = new pg.Pool(process.env.DATABASE_URL);
+  server.app.db = new pg.Pool({ database: 'abase_test' });
```

Comment:
@SimonLab you're right, it doesn't work with a DATABASE_URL. I'm not sure we need a pool anymore anyway. All I want is a pg client in the plugin registration, and clients accessible from the handlers (for which we could use `hapi-postgres-connection` again).